dataset/src/main/scala/frameless/ops/GroupByOps.scala

```diff
@@ -82,6 +82,11 @@ class GroupedByManyOps[T, TK <: HList, K <: HList, KT](
   private def retainGroupColumns: Boolean = {
     self.dataset.sqlContext.getConf("spark.sql.retainGroupColumns", "true").toBoolean
   }
+
+  def pivot[P: CatalystPivotable, Values <: HList](pivotColumn: TypedColumn[T, P],
+                                values: Values)(
```

Comment:
Also I think we forgot the scaladoc in this file, but that could be for a subsequent PR.